src/axom/quest/CMakeLists.txt

```diff
@@ -105,6 +105,20 @@ if(CONDUIT_FOUND AND AXOM_ENABLE_MPI)
                                  conduit::conduit_mpi)
 endif()
 
+blt_list_append(
+    TO       quest_headers
+    ELEMENTS MarchingCubes.hpp detail/MarchingCubesImpl.hpp
+    IF       CONDUIT_FOUND
+               )
+
+blt_list_append(
+    TO       quest_sources
+    ELEMENTS MarchingCubes.cpp
+    IF       CONDUIT_FOUND
+               )
+
+blt_list_append( TO quest_depends_on ELEMENTS conduit::conduit IF CONDUIT_FOUND )
+
```

Comment:
This could all be combined with the above if

Reply:
Doesn't that only works if you're appending to the same variable?  I have 3 different variables, so 3 calls.

Reply:
I wasn't as careful with this comment as i should have been. I missed the AXOM_ENABLE_MPI in the if.

This is fine.